src/lib.rs

```diff
@@ -140,9 +141,104 @@ fn spawn_in_pool(jobs: Arc<Mutex<Receiver<Thunk>>>) {
     });
 }
 
+/// A scoped thread pool used to execute functions in parallel.
+///
+/// `ScopedThreadPool` is different from `TaskPool` in that:
+/// * It propagates panics that occur in the worker threads.
```

Comment:
This propagation will only happen on `drop` of the pool though (which may want to be mentioned)